### DISCUSSION TITLE
How to create a pull request?

### DISCUSSION THREAD

**Discussion by shaiarmis** [readmeio/rdme#578] [Q&A] [answered]
I have an idea for a new feature for the CLI, and I actually already have a local version of the change (including a relevant test).
I'd like to push a new branch and then open a PR, but I get the following error:

```
ERROR: Permission to readmeio/rdme.git denied to shaiarmis.
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
```

When I go the the [contribution guidelines](https://github.com/readmeio/rdme/blob/main/CONTRIBUTING.md), I don't see anything permissions-related, so I wonder what I should do.

Please advise :)

**Comment by domharrington** [comment, marked as answer]
Hey! You should fork the repository first, so it's on your github under https://github.com/shaiarmis/rdme, then you can push branches up to your fork. Once you have a branch pushed, you can create a new pull request against our repository and we'll review/discuss it there. 

More info:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#changing-the-branch-range-and-destination-repository

Looking forward to seeing your change!

**Reply by shaiarmis** [reply]
Awesome, thanks for the quick and informative reply :)

**Reply by shaiarmis** [reply]
Here's the [PR](https://github.com/readmeio/rdme/pull/579) :)
